pkg/trie/triedb/node.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package triedb
+
+import (
+	"bytes"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/pkg/trie/db"
+	"github.com/ChainSafe/gossamer/pkg/trie/triedb/codec"
+)
+
+type StorageHandle struct{ int }
```

Comment:
This name doesn't really help me figure out what this does, maybe there is a more appropriate name or can add a description above

Reply:
I'll add a comment while thinking in a better name 😄